### PR TITLE
Back to business

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by maxulysse** [comment]
True, but at the moment, not sure if the next release will be `3.0.1` or `3.1`, so I prefered to stay with `3.0` for now

**Review by asp8200 — APPROVED** [review]
LGTM

**Review by jfy133 — APPROVED** [review]
for the submway

I would still recommend rrotating the nf-core/sarek logo and image title to normal Left to right, you can move the legend to right justified.

But otherwise everything looks OK

You have lots of white space you cna use, and the current 90 rotation is dfficult to read

**Comment by maxulysse** [comment]
> You have lots of white space you cna use, and the current 90 rotation is dfficult to read

I do like that actually

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@
...
 
-<!-- On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/sarek/results). -->
+On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/sarek/results).
```
Comment (FriederikeHanssen) [inline README.md:26]: 🤦‍♀️ well there is always something to forget

**Pull request merged**